### ISSUE TITLE
[0.x] Unknown event (not part of the documentation) received when simulcast enabled

### ISSUE THREAD

**Issue by application-developer-DA** [meetecho/janus-gateway#2889]
**What version of Janus is this happening on?**
0.11.6 `fd589de767afbd82ca4d6ba2ff15700f8cf233f6`

**Have you tested a more recent version of Janus too?**
No (related to the docs), rather than to the specific version of Janus.

**Was this working before?**
The issue was not noticed before we started to use simulcast.

**Is there a gdb or libasan trace of the issue?**
Not applicable.

**Additional context**
Documentation is missing. The given event is not described on the [VideoRoom Plugin page](https://janus-legacy.conf.meetecho.com/docs/videoroom.html) and it's not clear how to interpret it and how to react to it.

The event body:
```
{
	"janus": "event",
	"session_id": 5173385215719922,
	"sender": 561907403993444,
	"plugindata": {
		"plugin": "janus.plugin.videoroom",
		"data": {
			"videoroom": "event",
			"room": 986866600377088,
			"substream": 0
		}
	}
}
```

**Comment by lminiero** [comment]
That's an event telling you which substream is currently selected on the subscription. Closing as not a code issue.

**Comment by application-developer-DA** [comment]
Thanks, @lminiero for the clarification. Should the issue be created in a different format (e.g. as here https://github.com/meetecho/janus-gateway/issues/2576), so that it is closed as soon as the documentation gets updated? (rationale: when it's closed due to the commit pushed to the branch that fixes the issue, it's clear for everyone that the meaning and the details of the previous undocumented part of the API became more transparent and so the one can rely on this information)
